src/styles/index.ts

```diff
@@ -3674,6 +3674,22 @@ const styles = (theme: ThemeColors) =>
             ...wordBreak.breakWord,
         },
 
+        quickActionTooltipWrapper: {
+            backgroundColor: colors.green100,
+        },
+
+        quickActionTooltipTitle: {
+            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE_BOLD,
+            fontWeight: FontUtils.fontWeight.bold,
+            fontSize: variables.fontSizeNormal,
+            lineHeight: variables.lineHeightXLarge,
+            color: colors.green500,
```

Comment:
Same comment as above, we should make these theme-friendly color values.

Reply:
>If this color does not differ between the themes, should we make them theme-friendly?

My vote is yes, in case we ever add a 3rd theme or make these theme dependent in the future.